.gitlab-ci.yml

```diff
@@ -71,15 +71,34 @@ check-line-width:
   allow_failure:                   false
 
 
+cargo-audit:
+  stage:                           test
+  <<:                              *compiler_info
+  script:
+    - cargo audit
+  allow_failure:                   true # failed cargo audit shouldn't prevent a PR from being merged
+  tags:
+    - linux-docker
+
+
+test-build-subkey:
+  stage:                           test
+  <<:                              *compiler_info
+  script:
+    - cd ./subkey
+    - time cargo build --release
```

Comment:
We don't require a release build. This should also reduce the pipeline time.

Reply:
ok